Sito/php/dbaccess.php

```diff
@@ -72,4 +341,98 @@ public function getPersonaggi()
+            <input type="text" id="provincia" name="provincia" value="Padova" disabled="disabled"/>
+            <label for="stato">Stato: </label>
+            <input type="text" id="stato" name="stato" value="Italia" disabled="disabled"/>
+            <label for="tel">Numero di telefono: </label>
+            <input type="tel" id="tel" name="tel" />
+
+            <input class="defaultButton" type="submit" name="dati_spedizione" value="Salva"/> <!--Submit legato solo alle informazioni di spedizione-->
+        	</fieldset>';
+}
+
+function getFormPagamento(){
+    return '<fieldset>
+            <legend id="ip">Informazioni di pagamento: </legend>
+                <label for="intestatario_carta">Intestatario carta: </label>
+                <input type="text" name="intestatario_carta" id="intestatario_carta" value="" />
+                <label for="num_carta">Numero carta: </label>
+                <input type="text" name="num_carta" id="num_carta" />
+            <select name="mese_scad">
+                <option>- Mese -</option>
+                <option value="01">January</option>
+                <option value="03">March</option>
+                <option value="04">April</option>
+                <option value="05">May</option>
+                <option value="06">June</option>
+                <option value="07">July</option>
+                <option value="08">August</option>
+                <option value="09">September</option>
+                <option value="10">October</option>
+                <option value="11">November</option>
+                <option value="12">December</option>
+        </select>
+        <select name="anno_scad">
+                <option>- Anno -</option>
+                <option value="2019">2019</option>
+                <option value="2020">2020</option>
+                <option value="2021">2021</option>
+                <option value="2022">2022</option>
+                <option value="2023">2023</option>
+                <option value="2024">2024</option>
+                <option value="2025">2025</option>
+                <option value="2026">2026</option>
+                <option value="2027">2027</option>
+                <option value="2028">2028</option>
+                <option value="2029">2028</option>
+                <option value="2030">2030</option>
+                <option value="2031">2031</option>
+                <option value="2032">2032</option>
+                <option value="2033">2033</option>
+                <option value="2034">2034</option>
+                <option value="2035">2035</option>
+                <option value="2036">2036</option>
+                <option value="2037">2037</option>
+                <option value="2038">2038</option>
+                <option value="2039">2039</option>
+            </select>
+
+            <input class="defaultButton" type="submit" name="dati_pagamento" value="Salva">
+        </fieldset>';
+}
+
```

Comment:
Queste funzioni di get sono specifiche (di sicuro) per la tua pagina quindi non ha senso che le scarichino tutti anche perchè non hanno niente a che fare con il database. Io le metterei direttamente nella pagina php come aveva fatto il prof in laboratorio.

Reply:
Si ma ho bisogno delle funzioni in entrambe le pagine, come faccio a metterle  nella pagina di gestione_profilo_utente e usarle in dbaccess.php?

Reply:
Non credo si possa. Ma tu non devi comunque avere la tua logica della pagina sul file db_access. Qui ottieni solo le informazioni è nell'altra che decidi cosa farne.